### DISCUSSION TITLE
Downloading this vault?

### DISCUSSION THREAD

**Discussion by BricoleurSoul** [obsidian-community/obsidian-hub#15] [Q&A] [answered]
Re: 

> To contribute, please **download the latest version of this vault and open it in Obsidian**. This is important to avoid duplicating content, and allows you to use the autocomplete when linking to other notes, searching and following our content guidelines

Is there a time frame in which this capability is expected?  Since the announcement of the Hub there has been no provision to do so from the <code> tab.

This is a wonderful initiative! and Hope that creative community members will contribute as envisioned but it seems to me that this piece must be in sooner rather than later if critical mass is to be achieved!

Thanks for all the work that has already been completed here.

**Comment by argenos** [comment, marked as answer]
Hey @BricoleurSoul! I've just added a release  so the link should be working now. That's for last week's version, so a lot of things have changed since! To download the vault's latest version, you can use this button: 
![image](https://user-images.githubusercontent.com/5426039/135640698-01816184-c558-4949-b634-a97903de8968.png)

For convenience, here's the link to download the [most recent contents](https://github.com/obsidian-community/obsidian-hub/archive/refs/heads/main.zip) any time (that is, everything in the `main` branch).
